plugins/removeformat/plugin.js

```diff
@@ -35,6 +35,11 @@ CKEDITOR.plugins.removeformat = {
 					range;
 
 				while ( ( range = iterator.getNextRange() ) ) {
+					// Prevent of removing `span` element with `lang` and `dir` attribute (#779).
+					editor.addRemoveFormatFilter( function( element ) {
```

Comment:
`addRemoveFormatFilter` method is available only with [removeFormat](https://docs.ckeditor.com/source/plugin77.html#CKEDITOR-editor-method-addRemoveFormatFilter) plugin, so that's the reason why I put it here. What if `removeFormat` plugin will be removed?